src/main/resources/log4j2.xml

```diff
@@ -28,10 +28,17 @@
                 <SizeBasedTriggeringPolicy size="20 MB"/>
             </Policies>
         </RollingRandomAccessFile>
+
+        <File name="HTMLAppender"
```

Comment:
I think this would make more sense to use a `RollingRandomAccessFile` appender similar to the one above, though I think the triggering policy might make more sense to use `TimeBasedTriggeringPolicy` using log file pattern of something like `filePattern="${AUDIT_LOG_DIRECTORY}/audit-%d{yyyy-MM-dd}.log"` and leave `append` as the default value. Then you could also add the `AUDIT_LOG_DIRECTORY` property that would be separate from the `AUDIT_LOG_DESTINATION` property (which can reuse this new directory property of course).

Updating the policies for the audit json log file to match the same daily log file pattern would be great, too, though we can do that in a separate PR if this one is getting too broad.

Reply:
That filePattern is for the audit.log file, though you can change it to .html for this particular one using the right directory.